src/Controller/ReportController.php

```diff
@@ -768,7 +768,7 @@ private function loopThroughSuite(int $executionId, stdClass $suite, ?int $paren
             if (isset($test->context)) {
                 try {
                     $identifier_data = json_decode($test->context);
-                    $identifier = $identifier_data->value;
+                    $identifier = is_array($identifier_data) ? $identifier_data[0]->value : $identifier_data->valu;
```

Comment:
```suggestion
                    $identifier = is_array($identifier_data) ? $identifier_data[0]->value : $identifier_data->value;
```
Isn't there a typo here ?

Reply:
:+1: